src/template.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To Do list</title>
+</head>
+<body>
+    <section id="TDBorder">
+    <p id="ToDo">Today's To Do List</p>
+    <input id="inputid" type="text" required placeholder="Add to your To Do list"><img id="enterid" src="../src/assets/enter.png">
+    <div id="list"></div>
+    <p id="clear">Clear all completed</p>
```

Comment:
- [ ] Kindly center the "clear completed button" as in the design.

- [ ] Add some left padding to the task description to provide some space between the checkbox and the description.

  your button | design button
  :---:|:---:
  ![sgye](https://user-images.githubusercontent.com/81468675/207851214-f9a46739-b4e5-4656-9233-c6e6bea0f25b.jpg) | ![kdkd](https://user-images.githubusercontent.com/81468675/207851222-309914ea-8cd0-4eb9-89bb-62fede131a17.jpg)